xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
There's basically no case where a circular dependency isn't broken. If you actually care about our architecture as you claim then you wouldn't accept this as a design going forward. 

This adds to the mess we're trying to clean up and get away from. 

I don't claim to be a maintainer of AE but I care about our overall code quality and try to fix/educate wherever possible.